<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>n_neighbours:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="train">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>480</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Train</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mean Square Error:</string>
    </property>
   </widget>
   <widget class="QLabel" name="mae">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>aaa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="visualize">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>360</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Visualize Boundary</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Accuracy Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mean Absolute Error:</string>
    </property>
   </widget>
   <widget class="QLabel" name="mse">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>120</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>aaa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Root Mean Sq. Error:</string>
    </property>
   </widget>
   <widget class="QLabel" name="rmse">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>150</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>aaa</string>
    </property>
   </widget>
   <widget class="QComboBox" name="X_combo">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>340</y>
      <width>79</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="columns">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>271</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="train_size">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>aa</string>
    </property>
   </widget>
   <widget class="QLabel" name="test_size">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>aaa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>311</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Train Size:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Y_combo">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>380</y>
      <width>79</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Test Size:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="neighbours">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>Number of neighbors</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="test_data">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>0.1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="test_size_btn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="accuracy">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>180</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>aaa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Train Size:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="conf_mat">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>450</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Confusion Matrix</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Target : </string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>20</width>
      <height>581</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="report">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>451</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Classification Report:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Light</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(11, 34, 57);</string>
    </property>
    <property name="text">
     <string>K-Nearest Neighbors Classifier</string>
    </property>
   </widget>
   <widget class="QLabel" name="target_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Dropout or Graduate&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="predict">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>480</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Predict</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_link">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
